modules/default-requirements-secured-cluster-services.adoc

```diff
@@ -11,11 +11,13 @@ Secured cluster services contain the following components:
 * Sensor
 * Admission controller
 * Collector
+* Scanner (optional)
+* Scanner V4 (optional)
 
 [id="default-requirements-secured-cluster-services-sensor_{context}"]
 == Sensor
 
-Sensor monitors your Kubernetes and {ocp} clusters. These services currently deploy in a single deployment, which handles interactions with the Kubernetes API and coordinates with Collector.
+Sensor monitors your Kubernetes and {ocp} clusters. These services currently deploy in a single deployment, which handles interactions with the Kubernetes API and coordinates with the other RHACS components.
```

Comment:
@kcarmichael08 I think there is some kind of template like `{ocp}` but for RHACS. Do you know what it is, so I can replace my hardcoded `RHACS` with it?

Reply:
It's `{product-title-short}`. You can find all of the variables in the common attributes file in the modules folder: https://github.com/openshift/openshift-docs/blob/rhacs-docs-main/modules/common-attributes.adoc